src/tunnel.cc

```diff
@@ -188,13 +189,15 @@ class TunnelStateData: public PeerSelectionInitiator
     time_t startTime; ///< object creation time, before any peer selection/connection attempts
     ResolvedPeersPointer destinations; ///< paths for forwarding the request
     bool destinationsFound; ///< At least one candidate path found
-    /// whether another destination may be still attempted if the TCP connection
-    /// was unexpectedly closed
-    bool retriable;
```

Comment:
I renamed this to banRetries because we have to modify most of its code. This change improves symmetry with FwdState (where a very similar field is also "negative"), provides better debugging, and avoids creating a somewhat unusual true/set-at-construction flag. Said that, it is slightly out of scope and can be undone if others insist on keeping this field  the same in this PR.